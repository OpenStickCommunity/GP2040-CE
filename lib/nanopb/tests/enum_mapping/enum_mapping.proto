syntax = "proto3";

enum TestEnum1
{
    A = 0;
    B = 2;
    C = -3;
}

enum TestEnum2
{
    X = 0;
    Y = 5;
}

message TestMsg
{
    TestEnum1 test1 = 1;
    
    oneof oneof1
    {
        TestEnum2 test2 = 2;
    }
}


