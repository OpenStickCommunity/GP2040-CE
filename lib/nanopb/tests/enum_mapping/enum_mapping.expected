define TestMsg_test1_ENUMTYPE TestEnum1
define TestMsg_oneof1_test2_ENUMTYPE TestEnum2

